docs/bin/import-tutorial-readme.sh

```diff
@@ -8,9 +8,55 @@
 # have to hand-edit the files after this import!
 # ----------------------------------------------------------------------
 
-# Upate for your system
-GEOMESA_PATH=/path/to/geomesa
-TUTORIAL_PATH=/path/to/geomesa-tutorials
+# default paths
+# may be overridden by setting variables or by command line switches
+# (will try to guess $GEOMESA relative to the script path)
+GEOMESA=${GEOMESA:-$(readlink -f "$(dirname ${BASH_SOURCE[0]})/../..")}
+GEOMESA_TUTORIALS=${GEOMESA_TUTORIALS:-/path/to/geomesa-tutorials}
+
+function usage {
+    echo "$0: imports several README.md files from the GeoMesa tutorials distribution,"
+    echo "using pandoc to convert Markdown to RST."
+    echo
+    echo "usage: $0 [options] [<tutorials-to-copy>]"
+    echo
+    echo "If <tutorials-to-copy> is omitted, all tutorials will be copied."
+    echo
+    echo "options:"
+    echo "  --gm <dir>     set \$GEOMESA [$GEOMESA]"
+    echo "  --tut <dir>    set \$GEOMESA_TUTORIALS [$GEOMESA_TUTORIALS]"
+}
+
+# parse options
+while :
+do
+   case "$1" in
+       --gm)
+           GEOMESA="$2"
+           shift 2
+           ;;
+       --tut)
+           GEOMESA_TUTORIALS="$2"
+           shift 2
+           ;;
+       -h | --help)
+           usage
+           exit 1
+           ;;
+       --) # manual end of options
+           break
+           ;;
+       -*)
+           echo "ERROR: unknown option $1" >&2
+           exit 2
+           ;;
+       *) # end of options
+           break
+           ;;
+   esac
+done
+
+#
 if [ "$#" -eq 0 ] ; then
     TUTORIALS="
 geomesa-examples-authorizations
```

Comment:
Oh, nice!

It doesn't look like the geomesa-quickstart-nifi PR has been merged into the master of geomesa-tutorials yet; I'll review that after this.